sdcard/rootfs/kube-systemd/etc/kubernetes/dropins/docker-flannel.conf

```diff
@@ -15,7 +15,8 @@ ExecStart=/usr/bin/docker daemon \
     --insecure-registry=registry.kube-system.svc.cluster.local:5000 \
     --insecure-registry=10.0.0.20:5000 \
     --insecure-registry=registry.kube-system:5000 \
-    --exec-opt native.cgroupdriver=cgroupfs
+    --exec-opt native.cgroupdriver=cgroupfs \
+    $DOCKER_STORAGE_OPTS
```

Comment:
This should be `DOCKER_OPTS` rather, as you can add any docker option. The environment env should be passed as is without any escaping ${ENV} is handled differently than $ENV by systemd.